funcx_sdk/funcx/tests/test_async.py

```diff
@@ -0,0 +1,63 @@
+import random
+
+
+def squared(x):
+    return x ** 2
+
+
+def delay_n(n):
+    import time
+
+    time.sleep(n)
+    return "hello"
+
+
+async def simple_task(fxc, endpoint):
+    squared_function = fxc.register_function(squared)
+    x = random.randint(0, 100)
+    task = fxc.run(x, endpoint_id=endpoint, function_id=squared_function)
+    result = await task
```

Comment:
can we do a asyncio.waitfor or any timeout style code here to bound the time we'll wait for results?

Reply:
Yes, `wait_for` should work I think